modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -209,12 +264,12 @@ You can query a vector index using the procedure {link-procedures-reference}#pro
 .Signature for `db.index.vector.queryNodes` to query a vector index
 [source,syntax,role="noheader",indent=0]
 ----
-db.index.vector.queryNodes(indexName :: STRING, numberOfNearestNeighbours :: INTEGER, query :: LIST<FLOAT>) :: (node :: NODE, score :: FLOAT)
+db.index.vector.queryNodes(indexName :: STRING, numberOfNearestNeighbours :: INTEGER, query :: LIST<INTEGER | FLOAT>) :: (node :: NODE, score :: FLOAT)
```

Comment:
I guess we still need to document the query relationship index procedure? (in this file as well as it was only added to the syntax section)

Reply:
Do we want an example of using it as well or are we happy with just listing it? (might be a question for docs team)

Reply:
working on it :sweat_smile: just pushed it so docs can have a look